src/OrchardCore.Modules/OrchardCore.Flows/Views/FlowPart.Edit.cshtml

```diff
@@ -13,6 +13,8 @@
     var htmlFieldPrefix = ViewContext.ViewData.TemplateInfo.HtmlFieldPrefix;
     var parentContentType = Model.FlowPart.ContentItem.ContentType;
     string partName = ((dynamic)Model).Metadata.Name;
+
+    Context.Features.Set(new OrchardCore.ContentManagement.IgnoreContentFeature());
```

Comment:
Probably it needs to be UnSet, i.e. Removed after the items have been rendered, or multiple content item renders, which are `unrelated to widgets or bags will be impacted.

We need a better name for it, it doesn't really indicate what it does.

`ContainedItemRendering` perhaps? Not feeling very inspired on naming today